src/ui/TreeView.js

```diff
@@ -50,11 +52,14 @@ class TreeView {
     return yo`<ul style=${this.cssList}>${children}</ul>`
   }
 
-  formatDataInternal (key, data, children, expand) {
+  formatDataInternal (key, data, children, properties, expand) {
+    var renderedProperties = Object.keys(properties).map((item) => {
```

Comment:
Is the main reason for the distinction between properties and children the way they are sorted? Because in general, this thing can also be achieved without having a second 'extract' function.